app/uk/gov/hmrc/debttransformationstub/controllers/ETMPController.scala

```diff
@@ -66,13 +66,10 @@ class ETMPController @Inject() (environment: Environment, cc: ControllerComponen
     val currentDate = LocalDate.now()
 
     val responseTemplate: String =
-      Using(Source.fromFile(file))(source => source.mkString)
-        .recoverWith {
-          case ex: Throwable =>
-            // Explain which file failed to be read.
-            Failure(new RuntimeException(s"Failed to read file: ${file.getPath}", ex))
-        }
-        .get // Can throw.
+      Using(Source.fromFile(file))(source => source.mkString).recoverWith { case ex: Throwable =>
+        // Explain which file failed to be read.
+        Failure(new RuntimeException(s"Failed to read file: ${file.getPath}", ex))
+      }.get // Can throw.
```

Comment:
Disappointingly, scalafmt made this harder to read.

In scalafmt version `3.0.0` they introduced [`newlines.selectChains = keep`](https://scalameta.org/scalafmt/docs/configuration.html#newlinesselectchains), which IMO is a lot better than the current behaviour. Something to keep in mind, perhaps?